Update ADOPTERS.md

ddonahuex opened the pull request:
Added nethopper.io to list of Adopters.

<!--
Thank you for helping to improve Crossplane!

Please read through https://git.io/fj2m9 if this is your first time opening a
Crossplane pull request. Find us in https://slack.crossplane.io/messages/dev if
you need any help contributing.
-->

### Description of your changes

<!--
Briefly describe what this pull request does. Be sure to direct your reviewers'
attention to anything that needs special consideration.

We love pull requests that resolve an open Crossplane issue. If yours does, you
can uncomment the below line to indicate which issue your PR fixes, for example
"Fixes #500":

-->
Fixes #

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested
I tested the md rendering before committing.

<!--
Before reviewers can be confident in the correctness of this pull request, it
needs to tested and shown to be correct. Briefly describe the testing that has
already been done or which is planned for this change.
-->

[contribution process]: https://git.io/fj2m9

jeanduplessis commented:
@ddonahuex would you mind signing your commit to comply with the [DCO check](https://probot.github.io/apps/dco/)?

jbw976 reviewed (approved):
Thank you very much for this addition and your support of the project @ddonahuex! 🤩

jbw976 commented:
We've got a few of these additions to the adopters list today, so we'll be taking them in the order that they were opened.

Unfortunately that means a small merge conflict to handle in the table while you're adding the DCO sign-off.  Thank you very much! :)

jbw976 commented:
I've also opened a PR that provides a bit more guidance around the DCO, so check that out @ddonahuex if you need a bit more context and example git commands 🙇 

#3716

ddonahuex commented:
Hi Jared,

Thanks for the info. The PR contains another company 'runwhen'. I'm not
clear what the ask is on my end. Has my original PR been rejected because
of DCO compliance? If so, should I make my changes and submit another PR?
Thanks,
Dan

On Thu, Feb 2, 2023 at 9:19 PM Jared Watts ***@***.***> wrote:

> I've also opened a PR that provides a bit more guidance around the DCO, so
> check that out @ddonahuex <https://github.com/ddonahuex> if you need a
> bit more context and example git commands 🙇
>
> #3716 <https://github.com/crossplane/crossplane/pull/3716>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/crossplane/crossplane/pull/3709#issuecomment-1414678916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7SUGGZQKJGM7HWV5CYUTTWVRTKDANCNFSM6AAAAAAUPNFCMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best,

Dan Donahue

***@***.***
603-475-9456)

ddonahuex commented:
Updated my DCO

ddonahuex commented:
I updated my DCO and added my testing comments to my original PR. Is there
anything else i need to do?


On Thu, Feb 2, 2023 at 9:19 PM Jared Watts ***@***.***> wrote:

> I've also opened a PR that provides a bit more guidance around the DCO, so
> check that out @ddonahuex <https://github.com/ddonahuex> if you need a
> bit more context and example git commands 🙇
>
> #3716 <https://github.com/crossplane/crossplane/pull/3716>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/crossplane/crossplane/pull/3709#issuecomment-1414678916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7SUGGZQKJGM7HWV5CYUTTWVRTKDANCNFSM6AAAAAAUPNFCMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best,

Dan Donahue

***@***.***
603-475-9456)

jbw976 commented:
Thanks for taking a look at the DCO @ddonahuex!  There are 2 separate things for us to resolve here:

1. The DCO is an automated check on every PR, so it prevents a PR from being merged until the DCO is signed-off and the check is green ✅ .  It's still red ❌  right now, did you already push the commit where you fixed it on your side?
1. Because this file is getting updated by multiple people today, there is a merge conflict to resolve. Another company, RunWhen, was added to the same place before your PR, so you essentially have to resolve this "conflict" and re-add your addition to the end of the list.
    1. There's a button I see on the PR that says "Resolve conflicts", do you also see this button?
    1. Let me know if you'd like me to try to resolve the conflicts on your behalf, happy to help with that and maybe save you a headache 😇 

![Screenshot 2023-02-02 at 6 43 01 PM](https://user-images.githubusercontent.com/4313439/216500019-a2c381d4-7f66-4da0-a4f0-683924a597fb.png)

ddonahuex commented:
I originally edited it directly on the github page using the web editor.
Sounds like I need to check it out, resolve the conflict and commit. Will
the conflict be in branch *patch-1* of ddonahuex/crossplane?


On Thu, Feb 2, 2023 at 9:43 PM Jared Watts ***@***.***> wrote:

> Thanks for taking a look at the DCO @ddonahuex
> <https://github.com/ddonahuex>! There are 2 separate things for us to
> resolve here:
>
>    1. The DCO is an automated check on every PR, so it prevents a PR from
>    being merged until the DCO is signed-off and the check is green ✅ .
>    It's still red ❌ right now, did you already push the commit where you
>    fixed it on your side?
>    2. Because this file is getting updated by multiple people today,
>    there is a merge conflict to resolve. Another company, RunWhen, was added
>    to the same place before your PR, so you essentially have to resolve this
>    "conflict" and re-add your addition to the end of the list.
>       1. There's a button I see on the PR that says "Resolve conflicts",
>       do you also see this button?
>       2. Let me know if you'd like me to try to resolve the conflicts on
>       your behalf, happy to help with that and maybe save you a headache
>       😇
>
> [image: Screenshot 2023-02-02 at 6 43 01 PM]
> <https://user-images.githubusercontent.com/4313439/216500019-a2c381d4-7f66-4da0-a4f0-683924a597fb.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/crossplane/crossplane/pull/3709#issuecomment-1414722320>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7SUGAJSZOWGVQ5HLXRPZ3WVRWFJANCNFSM6AAAAAAUPNFCMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best,

Dan Donahue

***@***.***
603-475-9456)

ddonahuex commented:
If you want to resolve, feel free. In the future I'll do it from the
command line instead of the web editor.

On Thu, Feb 2, 2023 at 9:43 PM Jared Watts ***@***.***> wrote:

> Thanks for taking a look at the DCO @ddonahuex
> <https://github.com/ddonahuex>! There are 2 separate things for us to
> resolve here:
>
>    1. The DCO is an automated check on every PR, so it prevents a PR from
>    being merged until the DCO is signed-off and the check is green ✅ .
>    It's still red ❌ right now, did you already push the commit where you
>    fixed it on your side?
>    2. Because this file is getting updated by multiple people today,
>    there is a merge conflict to resolve. Another company, RunWhen, was added
>    to the same place before your PR, so you essentially have to resolve this
>    "conflict" and re-add your addition to the end of the list.
>       1. There's a button I see on the PR that says "Resolve conflicts",
>       do you also see this button?
>       2. Let me know if you'd like me to try to resolve the conflicts on
>       your behalf, happy to help with that and maybe save you a headache
>       😇
>
> [image: Screenshot 2023-02-02 at 6 43 01 PM]
> <https://user-images.githubusercontent.com/4313439/216500019-a2c381d4-7f66-4da0-a4f0-683924a597fb.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/crossplane/crossplane/pull/3709#issuecomment-1414722320>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7SUGAJSZOWGVQ5HLXRPZ3WVRWFJANCNFSM6AAAAAAUPNFCMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best,

Dan Donahue

***@***.***
603-475-9456)

jbw976 commented:
awesome @ddonahuex, i really appreciate your willingness to collaborate here - `git` does make some workflows more challenging then you'd hope for 😊 

i was able to do a bit of surgery to update the branch, resolve the conflicts, and add the sign-off in one swoop.  Everything looks green ✅  now, so I'll go ahead and merge!

Thanks again for this contribution and your usage of the Crossplane project!! 🙇

ddonahuex commented:
Thank you very much, Jared!

On Thu, Feb 2, 2023 at 10:14 PM Jared Watts ***@***.***>
wrote:

> awesome @ddonahuex <https://github.com/ddonahuex>, i really appreciate
> your willingness to collaborate here - git does make some workflows more
> challenging then you'd hope for 😊
>
> i was able to do a bit of surgery to update the branch, resolve the
> conflicts, and add the sign-off in one swoop. Everything looks green ✅
> now, so I'll go ahead and merge!
>
> Thanks again for this contribution and your usage of the Crossplane
> project!! 🙇
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/crossplane/crossplane/pull/3709#issuecomment-1414774938>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7SUGCNRCAACFHAKNLHVXTWVRZ2TANCNFSM6AAAAAAUPNFCMY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Best,

Dan Donahue

***@***.***
603-475-9456)
